Microsoft.Research/Contracts/System.Windows.Forms/System.Windows.Forms.Application.cs

```diff
@@ -282,7 +283,10 @@ public static FormCollection OpenForms
     //
     // Returns:
     //     One of the System.Threading.ApartmentState values.
-    //public static ApartmentState OleRequired();
+    public static ApartmentState OleRequired()
```

Comment:
Could you please clarify what is the reason to uncommenting this? 

Any plans to add postconditions here?